dozer/cogs/info.py

```diff
@@ -153,33 +166,53 @@ async def rolemembers(self, ctx: DozerContext, role: discord.Role):
-
-        embed.set_thumbnail(url=guild.icon.url if guild.icon is not None else None)
-
-        embed.add_field(name='Created on', value=f"<t:{int(guild.created_at.timestamp())}:f>")
-        embed.add_field(name='Owner', value=guild.owner)
-        embed.add_field(name='Emoji', value=f"{static_emoji} static, {animated_emoji} animated")
-        embed.add_field(name='Roles', value=str(len(guild.roles) - 1))  # Remove @everyone
-        embed.add_field(name='Channels', value=str(len(guild.channels)))
-        embed.add_field(name='Nitro Boost Info', value=f'Level {ctx.guild.premium_tier}, '
-                                                       f'{ctx.guild.premium_subscription_count} booster(s), '
-                                                       f'{ctx.guild.filesize_limit // 1024 ** 2}MiB files, '
-                                                       f'{ctx.guild.bitrate_limit / 1000:0.1f}kbps voice')
-
-        await ctx.send(embed=embed)
+        e = discord.Embed(color = blurple)
+        e.set_thumbnail(url = guild.icon.url if guild.icon else None)
+        e.title = guild.name
+        e.description = f"{guild.member_count} members, {len(guild.channels)} channels, {len(guild.roles) - 1} roles"
+        e.add_field(name = 'ID', value = guild.id)
+        e.add_field(name = 'Created on', value = discord.utils.format_dt(guild.created_at))
+        e.add_field(name = 'Owner', value = guild.owner.mention)
+        e.add_field(name = 'Emoji', value = f"{static_emoji} static, {animated_emoji} animated")
+        e.add_field(name = 'Nitro Boost', value = f'Level {ctx.guild.premium_tier}, '
+                                                  f'{ctx.guild.premium_subscription_count} booster(s)\n'
+                                                  f'{ctx.guild.filesize_limit // 1024**2}MiB files, '
+                                                  f'{ctx.guild.bitrate_limit / 1000:0.1f}kbps voice')
+        await ctx.send(embed = e, ephemeral = True)
 
     guild.example_usage = """
     `{prefix}guild` - get information about this guild
     """
 
+    @commands.hybrid_command()
+    async def stats(self, ctx: DozerContext):
+        """Get current running internal/host stats for the bot"""
+        info = await ctx.bot.application_info()
+
+        frame = "\n".join(
+            map(lambda x: f"{str(x[0]):<24}{str(x[1])}", {
+                "Users:": len(ctx.bot.users),
+                "Channels:": len(list(ctx.bot.get_all_channels())),
+                "Servers:": len(ctx.bot.guilds),
+                "": "",
+                f"{' Host stats ':=^48}": "",
+                "Operating system:": os_name,
+                "Process uptime": str(datetime.timedelta(seconds = round(time.time() - startup_time)))
+            }.items()))
+        embed = discord.Embed(title = f"Stats for {info.name}", description = f"Bot owner: {info.owner.mention}```{frame}```", color = blurple)
+        await ctx.send(embed=embed, ephemeral = True)
```

Comment:
Not ephemeral

Reply:
removed, but this is one that probably should be set to ephemeral